### PR TITLE
Added several quality of life improvements

### PR DESCRIPTION
**Pull request by nebril**
Turning off selinux on slaves
kubectl on solar master node
using pigmej/pykube fork

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -180,6 +180,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           rsync__args: ["--verbose", "--archive", "--delete", "-z"]
         end
       end
+      config.vm.provision "shell", inline: "sudo /usr/sbin/setenforce 0"
```
Comment (pigmej) [inline Vagrantfile]: It should be done during in node.run

Comment (nebril) [inline Vagrantfile]: OK, will move.

**Comment by nhlfr** [comment]
looks good

File: fedora_cloud_base/1.0.0/actions/run.sh [hunk 2]
```diff
@@ -1 +1,2 @@
 sudo dnf install -y python python-dnf ansible libselinux-python
+sudo /usr/sbin/setenforce 0
```
Comment (pigmej) [inline fedora_cloud_base/1.0.0/actions/run.sh:2]: You should set it also for permanent in /etc/sysconfig/selinux BUT, let's not do it now, we need to migrate it from FC to Ubuntu (or different stuff)

File: deploy/deploy.sh [hunk 3]
```diff
@@ -1,11 +1,24 @@
 #! /bin/bash
 export SOLAR_CONFIG_OVERRIDE="/.solar_config_override"
 sudo pip install netaddr
+sudo pip install -I zmq
+
+wget https://storage.googleapis.com/kubernetes-release/release/v1.2.2/bin/linux/amd64/kubectl
```
Comment (pigmej) [inline deploy/deploy.sh:6]: Why ?

**Pull request merged**